schemas/external/repo/effectiveprivileges.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/base/effectiveprivileges",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Effective Privileges",
+  "description": "Schema for the Effective Privileges resource",
+  "type": "object",
+  "definitions": {
+    "effectiveprivileges": {
+      "type": "object",
+      "properties": {
+        "*": { "$ref": "#/definitions/privilege" }
```

Comment:
I know `*` is a valid JSON name, but it still feels odd to me. @kstreeter @lrosenthol WDYT?

Reply:
"*" was suggested by Felix. It does accurately described the property. Alternative might be "default".

Reply:
Ok, then we'll roll with it.